### PR TITLE
update strings

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

**Comment by timlinux** [comment]
LGTM - a few small corrections if you don't mind applying them...

**Comment by Gustry** [comment]
You forgot to send your comments

File: safe/utilities/keyword_io.py [hunk 1]
```diff
@@ -231,16 +232,17 @@ def copy_keywords(
...
-        except Exception, e:
+        except Exception as e:
             message = self.tr(
                 'Failed to copy keywords file from : \n%s\nto\n%s: %s' % (
                     source_layer.source(), new_destination, str(e)))
             raise Exception(message)
         return
 
-# methods below here should be considered private
+    # methods below here should be considered private
```
Comment (ismailsunni) [inline safe/utilities/keyword_io.py]: +1

Comment (Gustry) [inline safe/utilities/keyword_io.py]: We are already doing that. I think this one is legacy and I changed the indent only

**Comment by Gustry** [comment]
We need to be careful with this submodule. Thanks, fixed

**Pull request merged**